Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM jupyter/base-notebook:a07573d685a4
+# Built from... https://hub.docker.com/r/jupyter/base-notebook/
+#               https://github.com/jupyter/docker-stacks/blob/master/base-notebook/Dockerfile
+
+COPY cs3api4lab /opt/cs3/cs3api4lab
+COPY src /opt/cs3/src
+COPY style /opt/cs3/style
+COPY jupyter-config/cs3api4lab.json /opt/cs3/jupyter-config/cs3api4lab.json
+COPY setup.py /opt/cs3/setup.py
+COPY README.md /opt/cs3/README.md
+COPY package.json /opt/cs3/package.json
+COPY package-lock.json /opt/cs3/package-lock.json
+COPY yarn.lock /opt/cs3/yarn.lock
+COPY tsconfig.json /opt/cs3/tsconfig.json
+COPY pyproject.toml /opt/cs3/pyproject.toml
+
+USER root
+
+RUN cd /opt/cs3 && \
+	python -m pip install --upgrade pip && \
+	pip install --no-cache-dir jupyter_packaging && \
+	pip install --no-cache-dir -e . && \
+    jupyter lab --generate-config -y && \
+	jupyter serverextension enable --py cs3api4lab --sys-prefix && \
+	jlpm && \
+	jlpm build && \
+	jupyter labextension install . && \
+	jlpm build && \
+    jupyter lab build -y && \
+    jupyter lab clean -y && \
+	npm cache clean --force && \
+    rm -rf "/home/${NB_USER}/.cache/yarn" && \
+    rm -rf "/home/${NB_USER}/.node-gyp" && \
+    fix-permissions "${CONDA_DIR}" && \
+    fix-permissions "/opt/cs3" && \
+    fix-permissions "/home/${NB_USER}" && \
+	sed -i 's/#c.NotebookApp.contents_manager_class/c.NotebookApp.contents_manager_class/g' /home/${NB_USER}/.jupyter/jupyter_notebook_config.py && \
+	sed -i 's/notebook.services.contents.largefilemanager.LargeFileManager/cs3api4lab.api.cs3apismanager.CS3APIsManager/g' /home/${NB_USER}/.jupyter/jupyter_notebook_config.py
```

Comment:
This PR includes changes in README and mentions our CS3APIsManager implementation as far as I can see?

Reply:
I'm just asking why you need to point to `cs3api4lab.api.cs3apismanager.CS3APIsManager` when this looks to be included in `__init__` already, so you should be able to just do `cs3api4lab.CS3APIsManager`, right?

Reply:
We need to replace the default content class in JupyterLab (LargeFileManager) with our content class implementation. If we don't change the config file, JupyterLab load extension but don't replace the default content class.

Reply:
Guys, I'm just saying that since we do `from cs3api4lab.api.cs3apismanager import CS3APIsManager` inside `cs3api4lab/__init__.py`,  we don't need to do `c.NotebookApp.contents_manager_class = cs3api4lab.api.cs3apismanager.CS3APIsManager` because `c.NotebookApp.contents_manager_class = cs3api4lab.CS3APIsManager` is enough.
Is enough, simpler, hides the internal organization of the code and we don't need to change the README.

Reply:
Ok, got it